net/veth.go

```diff
@@ -49,6 +49,13 @@ func CreateAndAttachVeth(name, peerName, bridgeName string, mtu int, keepTXOn bo
 	if err := bridgeType.attach(veth); err != nil {
 		return cleanup("attaching veth %q to %q: %s", name, bridgeName, err)
 	}
+	// No ipv6 router advertisments please
+	if err := sysctl(procPath, "net/ipv6/conf/"+name+"/accept_ra", "0"); err != nil {
```

Comment:
I'm unsure whether `sysctl()` adds the leading `/` or not. Assuming procPath=`/proc`, i'm assuming sysctl uses filepath.join().

Reply:
You can see it here: https://github.com/weaveworks/weave/blob/6ad3e69faf9c88425db559cc8bf51c0a9f83c5e1/net/arp.go#L23

Reply:
i validated that this is the right path: `/proc/sys/net/ipv6/conf/INTERFACENAME/accept_ra`
from [here](https://sysctl-explorer.net/net/ipv6/accept_ra/)